### ISSUE TITLE
reimplement community header

### ISSUE THREAD

**Issue by BalogunofAfrica** [status-im/status-react#20049]
Reimplement header component for community, with better scroll handling.

Figma link:
- https://www.figma.com/design/h9wo4GipgZURbqqr1vShFN/Communities-for-Mobile?node-id=1487-100184&t=AsdLiorWfMIMRivR-0

Things to note:
-  https://github.com/status-im/status-mobile/pull/19665#pullrequestreview-2014688183
- Blur should not be implemented on Android (There should be a variant for this, please confirm with designers)

**Comment by ilmotta** [comment]
@BalogunofAfrica @J-Son89 @cammellos do you remember the final call about blur on Android? I thought we would completely remove it for Android and that we got the green light from designers. Or am I wrong and we will continue to try to implement blur on Android and only remove it in places where we failed to accomplish Figma specs? If that's the case, it still sounds to me like an unnecessary cost in development given its low importance to Android users and how historically blur hasn't been Android's best friend in this repo.

**Comment by J-Son89** [comment]
No idea but for important design decisions like this we need it in a place where we can all point to and communicate out. Otherwise this debate will just keep resurfacing and wasting our resources

**Comment by ilmotta** [comment]
> No idea but for important design decisions like this we need it in a place where we can all point to and communicate out. Otherwise this debate will just keep resurfacing and wasting our resources

Fully agree. I'll try to check this with designers and get the final word written somewhere. I'm tired too of coming back to this with no final decision.

**Comment by cammellos** [comment]
> we will continue to try to implement blur on Android

This is definitely not the case, we don't have really realistically time for it, so that reason alone can shortcut the decison

**Comment by BalogunofAfrica** [comment]
@ilmotta I should have added the notes on Android in the description, done that now. 

This issue was created because asides the blur issue this PR https://github.com/status-im/status-mobile/pull/19665 tried to fix, @ulisesmac and I agreed that the header for community might need to be reworked. So this is just to keep track till we have free time for it.

**Comment by ilmotta** [comment]
> @ilmotta I should have added the notes on Android in the description, done that now.
> 
> This issue was created because asides the blur issue this PR #19665 tried to fix, @ulisesmac and I agreed that the header for community might need to be reworked. So this is just to keep track till we have free time for it.

Sorry for hijacking the issue with this blur discussion, but it felt right to try and stamp a final decision about blur on Android that was still floating around (as you could see in Discord). Thanks for updating the requirements @BalogunofAfrica

**Comment by cammellos** [comment]
fyi https://github.com/status-im/status-mobile/issues/20066

**Comment by ilmotta** [comment]
I suggest we avoid fixing bugs in the community header component, and instead align with designers a simple solution that delivers the same value for users.

An example of a simpler solution is to eliminate stickiness entirely because the user can easily scroll back up to see the name of the community and press on `...` for more options. Many communities won't even have many channels.

cc @xAlisher an interesting problem to consider
cc @ulisesmac we were talking today about bugs in the header

Example of a community with fewer channels and weirdly behaving top header:

https://github.com/user-attachments/assets/50e2b502-5e69-4edb-999b-18343a8ccd32

**Comment by xAlisher** [comment]
I agree with @ilmotta. Let's eliminate any complexity if it's taking resources away from more important tasks.
